doc/Language/operators.pod6

```diff
@@ -3148,11 +3155,12 @@ knows about arithmetic and geometric sequences.
 If the endpoint is not C<*>, it's smartmatched against each generated
 element and the sequence is terminated when the smartmatch succeeded.
 For the C<...> operator, the final element is included, for the C<...^>
-operator it's excluded.
+and the C<^...^> operators it's excluded.
```

Comment:
I guess that, for symmetry, you should also mention the ^… operator, right?

Reply:
Hum... I think I see what you mean. What do you think about the commit f8fd3cc I pushed on this branch ?

Reply:
Essentially yes, but check out again my comments, sorry :-)

Reply:
no worries. :-)